app/controllers/spree/admin/payments_controller.rb

```diff
@@ -82,7 +82,7 @@ def object_params
            source_params = params.delete(:payment_source)[params[:payment][:payment_method_id]]
           params[:payment][:source_attributes] = source_params
         end
-        params[:payment]
+        params.require(:payment).permit(:amount, :payment_method_id, :source_attributes)
```

Comment:
Does this method still return the same output as before?

Reply:
I guess it must do.. :+1:

Reply:
yes, it's equivalent.